getting-started/android-with-appengine/README.md

```diff
@@ -11,13 +11,12 @@ The sample is called **Emojify**.
 
 Example of source and emojified images:
 
-<image src="screenshots/meetup.jpg" width="430px"/> <<image src="screenshots/emojified-meetup.jpg" width="430px"/>
+<image src="screenshots/meetup.jpg" width="430px"/> <image src="screenshots/emojified-meetup.jpg" width="430px"/>
 Picture taken at [DroidCon NYC Extended](https://dcnyc-extended-2018.splashthat.com/).
-            
-### Cloud Client Libraries
-The app integrates two Google Cloud Client Libraries:
-* [Google Cloud Storage](https://cloud.google.com/storage): hosts input and emojified images
-* [Google Cloud Vision](https://cloud.google.com/vision): used to perform face detection and image annotation on input image
+
+# Tutorial
+
+**This [codelab](https://g.co/codelabs/emojify) walks you through how to build and deploy Emojify.**
```

Comment:
I still would like to see this at the VERY top of the README, e.g.:

> This is a sample application to demonstrate a Kotlin backend which communicates with a mobile frontend.
Follow the [Extend an Android frontend with a Kotlin backend running on App Engine](https://codelabs.developers.google.com/codelabs/cloud-kotlin-emojify) codelab to build and run it yourself!